.vscode/settings.json

```diff
@@ -40,5 +40,8 @@
       "server": "localhost",
       "username": "sa"
     }
+  ],
+  "cSpell.words": [
+    "Pims"
   ]
```

Comment:
We shouldn't add settings related to 3rd-party plugins here. I do have the same plugin but most people won't. Please remove this and instead select the option "Add to USER dictionary" instead of "Add to WORKSPACE dictionary". That way your spelling preferences won't be stored in source control for this project. Thanks!

Reply:
Mixed feelings about this.  The settings won't cause an issue if you don't have the plugin.  Additionally if you do, it will make life better.